### PR TITLE
Ref #7867. Added Zephyr simulator 

### PR DESCRIPTION
**Pull request by pablogs9**
This PR adds a tutorial for Zephyr host simulation

### CODE REVIEW

File: _docs/tutorials/advanced/zephyr/zephyr_emulator/index.md [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+```bash
+# Run a micro-ROS agent
+ros2 run micro_ros_agent micro_ros_agent udp4 --port 8888
+```
+
+And run the Zephyr app in another command line (remember sourcing ROS 2 and micro-ROS installation): 
+
+```bash
+source /opt/ros/$ROS_DISTRO/setup.bash
+source install/local_setup.bash
```
Comment (julibert) [inline _docs/tutorials/advanced/zephyr/zephyr_emulator/index.md]:
```suggestion
source microros_ws/install/local_setup.bash
```

File: _docs/tutorials/advanced/zephyr/zephyr_emulator/index.md [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+source /opt/ros/$ROS_DISTRO/setup.bash
+
+# Subscribe to micro-ROS ping topic
+ros2 topic echo /microROS/ping
+```
+
+You should see the topic messages published by the Ping Pong node every 5 seconds:
+
+```
+pgarrido@pgarrido:~$ ros2 topic echo /microROS/ping
```
Comment (julibert) [inline _docs/tutorials/advanced/zephyr/zephyr_emulator/index.md]:
```suggestion
user@user:~$ ros2 topic echo /microROS/ping
```

File: _docs/tutorials/advanced/zephyr/zephyr_emulator/index.md [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+source /opt/ros/$ROS_DISTRO/setup.bash
+
+# Send a fake ping
+ros2 topic pub --once /microROS/ping std_msgs/msg/Header '{frame_id: "fake_ping"}'
+```
+
+Now, we should see on the ping subscriber our fake ping along with the board pings:
+
+```
+pgarrido@pgarrido:~$ ros2 topic echo /microROS/ping
```
Comment (julibert) [inline _docs/tutorials/advanced/zephyr/zephyr_emulator/index.md]:
```suggestion
user@user:~$ ros2 topic echo /microROS/ping
```

File: _docs/tutorials/advanced/zephyr/zephyr_emulator/index.md [hunk 4]
```diff
@@ -0,0 +1,402 @@
...
+stamp:
+  sec: 0
+  nanosec: 0
+frame_id: fake_ping
+---
+```
+
+## Multiple Ping Pong nodes
+
+One of the advantages of having an emulator is that you don't need to buy a bunch of hardware in order to test some multi-node micro-ROS apps. So, with the same micro-ROS agent of the last section, let's open four different command lines and run thw following on each:
```
Comment (julibert) [inline _docs/tutorials/advanced/zephyr/zephyr_emulator/index.md]:
```suggestion
One of the advantages of having an emulator is that you don't need to buy a bunch of hardware in order to test some multi-node micro-ROS apps. So, with the same micro-ROS agent of the last section, let's open four different command lines and run the following on each:
```

File: _docs/tutorials/advanced/freertos/freertos_getting_started/index.md [hunk 5]
```diff
@@ -72,6 +72,7 @@ cd firmware/freertos_apps/apps
 mkdir my_brand_new_app
 cd my_brand_new_app
 touch app.c app-colcon.meta
+cd ../../../
```
Comment (julibert) [inline _docs/tutorials/advanced/freertos/freertos_getting_started/index.md]: Maybe we could use `pushd` and `popd`

Comment (pablogs9) [reply _docs/tutorials/advanced/freertos/freertos_getting_started/index.md]: Done

**Pull request merged**